client/components/place/place-list/place-list.component.js

```diff
@@ -79,6 +80,16 @@ angular.module('placeList', ['filterMapType', 'popularTracks', 'ngAnimate'])
         };
         // -----END ADD Place-----
 
+        // ----START---- Function for add and remove html classes
+        ctrl.classAddRemove = function(tag, tagClass, add) {
```

Comment:
Also, you can remove `add` argument and use [.hasClass()](https://api.jquery.com/hasClass/) function